MatrixKit/Categories/NSBundle+MatrixKit.m

```diff
@@ -15,16 +15,17 @@
  */
 
 #import "NSBundle+MatrixKit.h"
+#import "MXKViewController.h"
 
 @implementation NSBundle (MatrixKit)
 
 static NSString *customLocalizedStringTableName = nil;
 
 + (NSBundle*)mxk_assetsBundle
 {
-    NSString *bundleResourcePath = [NSBundle mainBundle].resourcePath;
-    NSString *assetPath = [bundleResourcePath stringByAppendingPathComponent:@"MatrixKitAssets.bundle"];
-    return [NSBundle bundleWithPath:assetPath];
+    NSBundle * bundle = [NSBundle bundleForClass:[MXKViewController class]];
```

Comment:
I agree that the case where the retrieved bundle is the main one, must be handled differently.
But we should not return the main bundle, we want here the assets bundle.
` if (bundle ==  [NSBundle mainBundle])
{
    NSURL * url = [bundle URLForResource:@"MatrixKitAssets" withExtension:@"bundle"];
    return [NSBundle bundleWithURL:url];
} `

Could you please update your PR?